### DISCUSSION TITLE
OIDC for Azure AD unauthorised after success login

### DISCUSSION THREAD

**Discussion by encikbaek** [Budibase/budibase#6416] [Help]
Hi, 

Just installed budibase and currently try to integrate Azure AD for SSO. After follow step [budibasedoc](https://docs.budibase.com/docs/openid-connect), i manage to go into Microsoft login and after complete login its return this error  on browser {"message":"Unauthorized","status":403}. and if i refresh the browser, im receiving this error {"message":"Unable to verify authorization request state.","status":403}

**Comment by melohagan** [comment]
Hey @encikbaek 

Just to verify, did you read through this page in the docs as well? https://docs.budibase.com/docs/sso-with-azure-ad
